test/test_io.py

```diff
@@ -184,6 +184,13 @@ def test_read_video_pts_unit_sec(self):
             self.assertEqual(info["video_fps"], 5)
             self.assertEqual(info, {"video_fps": 5})
 
+    def test_read_video_pts_out_of_stream(self):
+        with temp_video(10, 300, 300, 5, lossless=True) as (f_name, data):
+            with self.assertRaises(BufferError):
+                lv, _, info = io.read_video(f_name, start_pts=5, pts_unit='sec')
```

Comment:
Just for the sake of completeness, could you add the same test for `pts_unit='pts'`. 

@fmassa can weigh in if this is really necessary, but given that we have tests for "sec" and "pts" in every case I think it would make sense

Reply:
Hi,
I'm afraid I don't know what proper values of pts would be. So if generated video is 10  frames at 5 fps video is 2 sec. A start time of out that would be 5 (for example). However I dunno how what the analogous reasoning would be for pts